src/lib/slider/slider.spec.ts

```diff
@@ -106,21 +110,21 @@ describe('MdSlider', () => {
     });
 
     it('should update the track fill on click', () => {
-      expect(trackFillElement.style.flexBasis).toBe('0%');
+      expect(trackFillElement.style.transform).toBe('scaleX(0)');
```

Comment:
Yeah, I realized I could do it all with transform and figured that was preferable.